docs/sinco/v3.x/1_getting_started/3_about_the_tutorials.md

```diff
@@ -0,0 +1,36 @@
+# About the Tutorials
+
+Most tutorials will follow the format below:
+
+**Before You Get Started**
+
+- This section has key things we feel you should know before you get started
+  with the tutorial.
+
+**Folder Structure End State**
+
+- This is what your project should look like at the end of the tutorial.
+
+**Steps**
+
+- A list of steps to get you from start to finish -- meeting the Folder
+  Structure End State.
+
+**Folder Structure End State**
+
+- This is what your project should look like at the end of the tutorial.
+
+**Verification**
+
+- After you follow the Steps section, you can use this Verification section to
+  test that your application is working.
+
+- Most, if not all verification steps will mention to test a file. You should
+  note that the commands these use do not restrict permissions, but it is
+  recommended that you do. The reason they don't is because what Sinco needs
+  access to is very dynamic, and very dependant on which OS and browser you are
+  using. You should refer to
+  [restricting permissions](tutorials/restricting-permissions) for more
```

Comment:
broken link. i couldn't figure out a way to use a "shorthand" URI, so i've been doing something like this in line:

```
/line/v1.x/tutorials/blah/blah
```

tedious, but i couldn't figure it out